src/transformers/models/clip/tokenization_clip.py

```diff
@@ -126,20 +126,34 @@ class BasicTokenizer(object):
         strip_accents (`bool`, *optional*):
             Whether or not to strip all accents. If this option is not specified, then it will be determined by the
             value for `lowercase` (as in the original BERT).
+        do_split_on_punc (`bool`, *optional*, defaults to `True`):
+            In some instances we want to skip the basic punctuation splitting so that later tokenization can capture
+            the full context of the words, such as contractions.
+        remove_control_chars (`bool`, *optional*, defaults to `True`):
+            Whether to allow or remove control characers, as some vocabs includes them.
```

Comment:
CLIP's vocab includes control chars, for example for [these two characters in the test](https://github.com/huggingface/transformers/blob/ae5fc2db87f2fb715c137c4823647f5b24c7f881/tests/models/clip/test_tokenization_clip.py#L104-L105)

Reply:
^update: it doesn't include them, those two in the test were just for matching the fast tokenizer which marks them as UNK